ui/src/vayla-design-lib/dialog/dialog.scss

```diff
@@ -163,3 +161,23 @@ $dialog-padding: $dialog-padding-vertical $dialog-padding-horizontal;
     overflow-y: auto;
   }
 }
+
+.dialog--narrow {
```

Comment:
Dialogeista on nyt 4 eri built-in-versiota. Mietin tuota `ultrawide`-nimeämistä, että olisiko joku tyyliin `dual-column` parempi, mutta ajattelin että ultrawide sopii paremmin sarjaan

Reply:
Joo. Nämä voisi toki nimetä myös tuon itse leveyden mukaan, sen sijaan että koittaa keksiä leveysarvoille nimiä. Tämäkin siis kyllä ok, mutta jos uutta dialogia tehdessä joutuu erikseen tarkastamaan että kuinkas leveän tällä tyylillä saankaan, olisi ehkä parempi vain laittaa leveys nimeen itseensä. Siis esim dialog--width-250, dialog--width-300, jne. Vähän inlineltähän sekin toki maistuu, mutta abstraktilla nimellä on arvoa vain jos abstraktio itsessään on hyödyllinen ja en osaa sanoa auttaako "narrow" vs "normal" käyttäjää paremmin kuin "w250" vs "w300".

Reply:
Kyselin ihmisten kantaa tähän slackissa tämän tiketin teon yhteydessä, ja nuo "narrow", "wide" jne. saivat enemmän kannatusta ihmisten keskuudessa. Itse kanssa tykkään näistä nykyisistä enemmän, sillä a) silloin ei tarvitse muuttaa luokkien nimiä jos päätetäänkin että narrow on vaikka 280 pikseliä 250 sijasta, ja b) suorien pikseliarvojen käyttöä ei CSS-tyyleissä ei pidetä nykyään enää muutenkaan suositeltavana koska responsiivisuusvaatimukset jne. Meille toki ei niin väliä kun tehdään käliä jota käytettäneen pienimmillään läppärin näytöltä, mutta kumminkin.